bindings/node/lib/clientEncryption.js

```diff
@@ -365,9 +365,6 @@ module.exports = function (modules) {
       if (options.keyId) {
         contextOptions.keyId = options.keyId.buffer;
       }
-      if (options.indexKeyId) {
-        contextOptions.indexKeyId = options.indexKeyId.buffer;
-      }
```

Comment:
I think you might want to remove this from the C++ source as well, since `contextOptinos` extends `options` and so could still contain an `indexKeyId` option

Reply:
This is removed from the C++ source now.